www/api/v4/index.php

```diff
@@ -249,7 +249,11 @@ function problems($args)
 			$api->createError("You cannot specify a primary ID both via /{id} and ?probid={id}");
 			return '';
 		}
-		$args['probid'] = rest_intid('problems', $args['__primary_key'], $cid);
+		$args['probids'] = array_map(function($probId) use ($cid) {
+			return rest_intid('problems', $probId, $cid);
```

Comment:
Please add same TODO comment as above to support multiple ids in rest_intid. Or at the function definition level.